src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
```

Comment:
```suggestion
        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
```
we can just use filename here